pkg/tsdb/cloudwatch/cloudwatch_test.go

```diff
@@ -246,4 +247,58 @@ func Test_executeLogAlertQuery(t *testing.T) {
 		assert.NoError(t, err)
 		assert.Equal(t, []string{"instance manager's region"}, sess.calledRegions)
 	})
+
+	t.Run("succeeds in returning data frames from minimal query results and stat group", func(t *testing.T) {
+		cli = fakeCWLogsClient{queryResults: cloudwatchlogs.GetQueryResultsOutput{
+			Results: [][]*cloudwatchlogs.ResultField{
+				{
+					{
+						Field: aws.String("some field"),
+						Value: aws.String("some value"),
+					},
+				},
+			},
+			Status: aws.String("Complete"),
+		}}
+		im := datasource.NewInstanceManager(func(s backend.DataSourceInstanceSettings) (instancemgmt.Instance, error) {
+			return datasourceInfo{}, nil
+		})
+		executor := newExecutor(im, newTestConfig(), &fakeSessionCache{})
+
+		resp, err := executor.QueryData(context.Background(), &backend.QueryDataRequest{
+			Headers:       map[string]string{"FromAlert": "some value"},
+			PluginContext: backend.PluginContext{DataSourceInstanceSettings: &backend.DataSourceInstanceSettings{}},
+			Queries: []backend.DataQuery{
+				{
+					TimeRange: backend.TimeRange{From: time.Unix(0, 0), To: time.Unix(1, 0)},
+					JSON: json.RawMessage(`{
+						"queryMode":    "Logs",
+						"region": "some region",
+						"statGroups": ["some stat group"]
```

Comment:
You have a typo here, it's `statsGroups` (and I guess that will have some impact in the result)